system/security/shared/basesecurity.hpp

```diff
@@ -303,6 +303,11 @@ class CBaseSecurityManager : implements ISecManager, public CInterface
         return NULL;
     }
 
+    bool logoutUser(ISecUser & user)
+    {
+        UNIMPLEMENTED;
```

Comment:
UNIMPLEMENTED implies that the code is incomplete...

If this is something that should never be called, use throwUnexpected()